examples/units/README.txt

```diff
@@ -1,5 +1,7 @@
 .. _units_examples:
 
+.. _units-examples-index:
```

Comment:
just FYI - sphinx gallery will auto-create these labels when it builds the docs. They follow the convention `.. _sphx_glr_path_to_file.py:`